packages/svelte-query/tests/createQueries/createQueries.test.ts

```diff
@@ -21,29 +21,22 @@ describe('createQueries', () => {
           queries: [
             {
               queryKey: ['key-1'],
-              queryFn: async () => {
-                await sleep(5)
-                return 'Success 1'
-              },
+              queryFn: () => sleep(10).then(() => 'Success 1'),
             },
             {
               queryKey: ['key-2'],
-              queryFn: async () => {
-                await sleep(5)
-                return 'Success 2'
-              },
+              queryFn: () => sleep(20).then(() => 'Success 2'),
```

Comment:
If queryFn's sleeping time is different 10ms, 20ms, it would be better to add test code for the case after just 10ms, not only for case after 20ms.